admin_guide/sdn_troubleshooting.adoc

```diff
@@ -630,6 +630,23 @@ provide the output from the debugging script)
 * Kube - declare as a LoadBalancer and something _you_ have to write does the rest
 * OS/AE - Both use the router
 
+[[tls-handshake-timeout]]
+=== TLS Handshake Timeout
+
+When a pod fails to deploy, check its docker log for a TLS handshake timeout:
+
+----
+$ docker log <container_id>
+...
+[...] couldn't get deployment [...] TLS handshake timeout
+...
+----
+
+This condition, and generally, errors in connection authentication,
```

Comment:
@liggitt Good point.  I've changed it to read "errors in establishing a secure connection".  WDYT?